{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "0beec5e1_df0af022",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 635
      },
      "writtenOn": "2023-01-30T11:58:57Z",
      "side": 1,
      "message": "recheck",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "6c550c78_928e95a6",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 141
      },
      "writtenOn": "2023-03-02T15:39:45Z",
      "side": 1,
      "message": "I was trying to understand what went wrong but I don\u0027t see a clear error, apart from a \n\nTagError: build python-cinder-tests-tempest-1.8.0-1.el8 already tagged (cloud8s-openstack-xena-candidate)\n\n",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "7546f28e_1b1c59cb",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 128
      },
      "writtenOn": "2023-03-03T08:25:31Z",
      "side": 1,
      "message": "recheck",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "337e657c_56e649a8",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 128
      },
      "writtenOn": "2023-03-03T08:25:56Z",
      "side": 1,
      "message": "race condition between the two jobs",
      "parentUuid": "6c550c78_928e95a6",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "c395b63e_5b297690",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 141
      },
      "writtenOn": "2023-03-03T13:07:57Z",
      "side": 1,
      "message": "recheck\n\nso should we retrigger it until it passes?",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "8fcbe8f6_033a13d6",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 128
      },
      "writtenOn": "2023-03-03T13:13:57Z",
      "side": 1,
      "message": "it should have passed this time, we must have some issue in ci scrit, i\u0027ll check.",
      "parentUuid": "c395b63e_5b297690",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "d853adb1_8161ed3e",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 128
      },
      "writtenOn": "2023-03-03T13:32:49Z",
      "side": 1,
      "message": "we are hitting a weird corner case because the original build https://cbs.centos.org/koji/taskinfo?taskID\u003d3197683 was reported as failed although the built was fine (probably cbs started two tag actions in parallel by mistake) and you can see that the tags are fine.\n\nThe ci job checks the status of the task which built the package and get it as failed so the job fails. TBH, i\u0027m not sure how to fix, because that check makes snese actually, so i will force the merge of this review.",
      "parentUuid": "8fcbe8f6_033a13d6",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "745218ea_4e806b75",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 141
      },
      "writtenOn": "2023-03-03T13:53:27Z",
      "side": 1,
      "message": "Weird, I remember other cases of tagging the same package version on different openstack releases and it usually worked.\n\nDoes this mean that also https://review.rdoproject.org/r/c/openstack/cinder-tempest-plugin-distgit/+/46776 (xena) and https://review.rdoproject.org/r/c/openstack/cinder-tempest-plugin-distgit/+/46775 (wallaby) require manual intervention?",
      "parentUuid": "d853adb1_8161ed3e",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "a2aeaf68_103e130b",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 128
      },
      "writtenOn": "2023-03-03T15:03:18Z",
      "side": 1,
      "message": "Yes, we do similar operations all the time. This case is because of a really unusual situation. We\u0027ll have to manually merge those too.",
      "parentUuid": "745218ea_4e806b75",
      "revId": "c6dce1fb34627c7739f68ea9b482d8e68b81d099",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    }
  ]
}